Firestore/Source/Public/FirebaseFirestore/FIRLoadBundleTask.h

```diff
@@ -18,6 +18,12 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+/**
+ * Represents the state of bundle loading tasks.
+ *
+ * Both `FIRLoadBundleTaskStateError` and `FIRLoadBundleTaskStateSuccess` are final states: task
+ * will abort or complete and there will be no more updates after they are reported.
```

Comment:
Nit: `will abort or complete` seems to indicate this actions will happen after these states are reported, but IIUC, the task will already be in one of these states. Consider something like `a task will be in either the aborted or the completed state and...`.

I also think `task` needs an article, but I'm not 100% sure if it's `a` or `the` (leaning towards `a`).